reporter/heap.go

```diff
@@ -8,9 +8,9 @@ func (h IntHeap) Len() int {
 	return len(h)
 }
 
-// Less tells whether elements at position j is less than the one at position i
+// Less tells whether elements at position i is less than the one at position j
 func (h IntHeap) Less(i, j int) bool {
-	return h[i] > h[j]
+	return h[i] < h[j]
```

Comment:
See my previous comment. Shouldn't it be reversed to achieve MaxHeap? https://golang.org/pkg/container/heap/#example__priorityQueue